command/update/flag/flag.go

```diff
@@ -29,9 +27,5 @@ func (f *Flag) Validate() error {
 		return microerror.MaskAnyf(invalidFlagsError, "provider kind must not be empty")
 	}
 
-	if len(f.Updater.Pod.Names) == 0 {
-		return microerror.MaskAnyf(invalidFlagsError, "pod names must not be empty")
-	}
-
```

Comment:
These flags got never used in production. I removed them completely because even the provider creation expected this configuration even though these information have not really been used for anything useful.